server/scripts/prime_database.sh

```diff
@@ -24,6 +24,10 @@ python3 manage.py collectstatic --noinput --clear
 # Load configurations
 python3 manage.py loaddata demo_config.yaml
 
+# Create superuser (password set in environment variable)
```

Comment:
question: Is this password set from DJANGO_SUPERUSER_PASSWORD? Is this a magically named environment variable that createsuperuser knows to use?

non-blocking: Could we update the comment to be more specific so folks without the PR context of seeing the provisioning/terraform/jobs.tf diff can trace what's happening? 

Potentially even better, expand the top-of-file comments to include examples of running this script outside of Cloud Run Jobs, where we can include any expected environment variables.

Reply:
1) Yes, it's a specific variable that's checked when creating the superuser https://docs.djangoproject.com/en/4.2/ref/django-admin/#createsuperuser

2) Will do.